api/modules/crudRefactorClean.js

```diff
@@ -0,0 +1,316 @@
+const sequelize = require('../models');
+
+const { models } = sequelize;
+const { 
+    User,
+    Course,
+    Topic, 
+    Language,
+    Feedback 
+} = models;
+
+
+module.exports = (function () {
+
+    //COURSES
+    const fetchCoursesById = (params) => {
```

Comment:
Since there will only ever be 1 course returned for a given `courseId`, renaming this to `fetchCourseById` would make a bit more logical sense

Same goes for all other `fetch*ById` functions below